phpBB/includes/mcp/mcp_queue.php

```diff
@@ -324,7 +324,33 @@ public function main($id, $mode)
 					'S_FIRST_POST'			=> ($post_info['topic_first_post_id'] == $post_id),
 
 					'U_LOOKUP_IP'			=> ($auth->acl_get('m_info', $post_info['forum_id'])) ? append_sid("{$phpbb_root_path}mcp.$phpEx", 'i=queue&amp;mode=approve_details&amp;f=' . $post_info['forum_id'] . '&amp;p=' . $post_id . '&amp;lookup=' . $post_info['poster_ip']) . '#ip' : '',
-				));
+				);
+
+				/**
+				* Alter post awaiting approval template before it is rendered
```

Comment:
Interestingly you didn't add the post data to the event variables though? Typo in the variable list?